api/v1alpha1/releaseplan_types.go

```diff
@@ -53,6 +55,15 @@ type ReleasePlanSpec struct {
 
 // ReleasePlanStatus defines the observed state of ReleasePlan.
 type ReleasePlanStatus struct {
+	// Conditions represent the latest available observations for the releasePlan
+	// +optional
+	Conditions []metav1.Condition `json:"conditions"`
+
+	// ReleasePlanAdmission contains the namespaced name of the releasePlanAdmission this ReleasePlan is
+	// matched to
+	// +kubebuilder:validation:Pattern=^[a-z0-9]([-a-z0-9]*[a-z0-9])?\/[a-z0-9]([-a-z0-9]*[a-z0-9])?$
+	// +optional
+	ReleasePlanAdmission string `json:"releasePlanAdmission,omitempty"`
```

Comment:
With our current code we could still have multiple ones, but this looks as it should. Just leaving a note here ;)